index.html

```diff
@@ -1885,6 +1885,7 @@ <h4 id="h-is-persistent-session-type" resource="#h-is-persistent-session-type"><
             </td></tr><tr><td class="prmName">initData</td><td class="prmType"><code>BufferSource</code></td><td class="prmNullFalse"><span role="img" aria-label="False">✘</span></td><td class="prmOptFalse"><span role="img" aria-label="False">✘</span></td><td class="prmDesc">
               <a href="#initialization-data">Initialization Data</a>
             </td></tr></tbody></table><div><em>Return type: </em><code>Promise&lt;void&gt;</code></div><p>When this method is invoked, the user agent must run the following steps:</p><ol class="method-algorithm">
+            <li><p>If the <a href="#session-close">Session Close</a> algorithm has been run on this object, return a promise rejected with a new <code><a href="https://heycam.github.io/webidl/#dfn-DOMException">DOMException</a></code> whose name is <code><a href="#dfn-InvalidStateError">InvalidStateError</a></code>.</p></li>
```

Comment:
In the existing cases, this step is after checking _callable_. We should try to be consistent by either running this step after checking _uninitialized_ OR making this the first step in all cases.

I'm leaning slightly towards the latter. What do you think?